doc/quick-install.md

```diff
@@ -114,13 +114,13 @@ If this all worked, then congratulations!  You have an operational Jitsi confere
 Jigasi is a server-side application acting as a gateway to Jitsi Meet conferences. It allows regular [SIP](https://en.wikipedia.org/wiki/Session_Initiation_Protocol) clients to join meetings and provides transcription capabilities.
 
 ```sh
-apt-get -y install jigasi
+sudo apt install jigasi
 ```
 or
 
 ```sh
 wget https://download.jitsi.org/unstable/jigasi_1.0-107_amd64.deb
-dpkg -i jigasi_1.0-107_amd64.deb
+sudo dpkg -i jigasi_1.0-107_amd64.deb
```

Comment:
Can you actually delete this "or" ection entirely? Jigasi is now on the staable repo so there is no need to take it from the unstable one.

Reply:
I can change that. I can run through the instructions again using the stable branch and test it, or change directly to https://download.jitsi.org/stable/jigasi_1.1-107-g6928850-1_amd64.deb?

Reply:
No, I mean remve everything after the "or". Just leave the existing `apt install jigasi` that's all there is to it now.

Reply:
I've applied the change to 77e0709: wget/dpkg removed from jagasi instructions

Reply:
Looking good!